pages/tips.md

```diff
@@ -24,6 +24,6 @@ _If you want to contribute, don't hesitate to send us a Pull Request with your t
 7. [Configuring Email with - Gmail and more]({{ site.url }}/tips/011_tip_configuring_email_in_jhipster.html)
 8. [Add a new Spring Social connector]({{ site.url }}/tips/012_tip_add_new_spring_social_connector.html)
 9. [Speed up the generator-jhipster]({{ site.url }}/tips/013_tip_speed_up_generator.html)
-10. [Using MariaDB]({{ site.url }}/tips/014_tip_using_mariadb.html)
```

Comment:
Did you also delete the MariaDB page? We need to do like https://github.com/jhipster/jhipster.github.io/blob/master/pages/openshift.md or we will have 404 errors which are bad for Google referencing